modules/subnets/main.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  subnets = {
+    for x in var.subnets :
+    "${x.subnet_region}/${x.subnet_name}" => x
+  }
+}
+
+resource "google_compute_subnetwork" "subnetwork" {
+  for_each                 = local.subnets
+  name                     = each.value.subnet_name
+  ip_cidr_range            = each.value.subnet_ip
+  region                   = each.value.subnet_region
+  private_ip_google_access = lookup(each.value, "subnet_private_access", "false")
```

Comment:
Can we default private_ip_google_access to true

Reply:
done